test/typescript/app.ts

```diff
@@ -57,7 +57,7 @@ export function fetchApp(organization: Organization) {
             }).catch(done)
         })
         test('Find all Authorized Apps', done => {
-            organization.app().findAllAuthorized()
+            organization.marketplace().findAllAuthorizedApps()
```

Comment:
Please review the naming convention. Should it be find() or findAll()? It's worth noting that find() is generally associated with retrieving all available content.